test/unittests/commands/xmlupload/test_resource_create_client.py

```diff
@@ -464,5 +466,21 @@ def test_make_iiif_uri_value_serialised() -> None:
     assert serialised == expected
 
 
+def test_make_boolean_value_no_permissions() -> None:
+    permission = {"prop-default": Permissions()}
+    xml_str = """
+        <resource label="foo_1_label" restype=":foo_1_type" id="foo_1_id">
+           <boolean-prop name=":isTrueOrFalse">
+            <boolean permissions="prop-default">true</boolean>
+        </boolean-prop>
+        </resource>
+        """
+    xmlresource = XMLResource(etree.fromstring(xml_str), "foo")
+    test_val: XMLValue = xmlresource.properties[0].values[0]
+    result = _make_boolean_value(test_val, BNode(), permission)
+
+    assert len(result) == 3
```

Comment:
Isn't it possible to do more thorough testing? For example like here: https://github.com/dasch-swiss/dsp-tools/blob/d818f5d885c0cc94d9169f16a3921f97ec0cfc18/test/unittests/commands/xmlupload/test_resource_create_client.py#L458